R/tools.R

```diff
@@ -139,11 +164,24 @@ rhino_style <- function() {
 #'   format_r("app/view")
 #' }
 #' @export
-format_r <- function(paths) {
+format_r <- function(paths, exclude_files = c("__init__\\.R")) {
```

Comment:
Do we need to exclude `__init__.R`? The recent upgrade that skips files where there is no `box::use` fixes the problem here.

Reply:
You're right. We can leave this NULL.